.github/workflows/NativeTemplateDev6.yml

```diff
@@ -0,0 +1,49 @@
+name: 'Custom Native Template Dev build'
+
+on:
+  workflow_dispatch:
+  schedule: 
+    - cron: '0 0 * * *'
```

Comment:
Sadly this can only be tested after merge, or you can set this in to be also pull_request for test purpose